ankerctl.py

```diff
@@ -434,5 +435,86 @@ def config_show(env):
             print(f"    p2p_hosts: {', '.join(p.p2p_hosts)}")
 
 
+@main.group("webserver", help="Built-in webserver support")
+@pass_env
+def webserver(env):
+    env.require_config()
+
+
+app = Flask(__name__)
+
+
+@app.get("/")
+def app_root():
+    prefix = "https://github.com/Ankermgmt"
+    return f"""<html>
+    <head><title>ankerctl web server</title></head>
+    <body>
+    <a href=\"{prefix}/ankermake-m5-protocol\">{prefix}/ankermake-m5-protocol</a>
+    <h1>Connecting prusaslicer:</h1>
+    <ul>
+    <li>Select "window" menu, "Printer Settings Tab"</li>
```

Comment:
This is good. 

As a user I'd like to see some more polish here (screenshots, ect...). I'm happy to help with that in a future PR

Reply:
Yes, this is very basic right now. More PRs absolutely welcome.